modules/dnn/src/net_impl.hpp

```diff
@@ -36,6 +36,32 @@ using std::string;
 
 typedef std::unordered_map<std::string, int64_t> NamesHash;
 
+
+// Zero'th version of PagedCacheManager
```

Comment:
Please add reference to paper or some description.

Reply:
I have updated the comment, hope that is sufficient